src/plugins/intel_npu/src/backend/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "openvino_npu_level_zero_backend")
+
+file(GLOB_RECURSE SOURCES *.cpp *.hpp *.h)
+source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR} FILES ${SOURCES})
+
+add_library(${TARGET_NAME} STATIC ${SOURCES})
+
+set_target_properties(${TARGET_NAME} PROPERTIES INTERPROCEDURAL_OPTIMIZATION_RELEASE ${ENABLE_LTO})
+ov_add_clang_format_target(${TARGET_NAME}_clang FOR_TARGETS ${TARGET_NAME})
+
+target_compile_definitions(${TARGET_NAME}
+    PRIVATE
+        IMPLEMENT_OPENVINO_RUNTIME_PLUGIN)
+
+target_include_directories(${TARGET_NAME}
+    PUBLIC
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+)
+
+target_link_libraries(${TARGET_NAME}
+    PRIVATE
+        openvino::npu_al
+        openvino_npu_zero_result_parser
+        $<BUILD_INTERFACE:ze_loader>
```

Comment:
it does not work this way. `BUILD_INTERFACE` cannot be used for libraries

Reply:
Removed the BUILD_INTERFACE, try to skip the target in export, but samples finally need to link to openvino::ze_loader to  pass link.